packages/migrate/src/transforms/2790-fix-transform.ts

```diff
@@ -17,7 +17,14 @@ import type RehearsalService from '../rehearsal-service';
 const OPTIONAL_TOKEN = '?';
 
 export default class FixTransform2790 extends FixTransform {
-  hint = `The operand of a 'delete' operator must be optional.`;
+  getHint = (replacements: { [key: string]: string }) => {
```

Comment:
You can clearly see the second approach just made code complicated - we have to "compute" hint in every getHint, instead of keep this logic outside (originally).